internal/keys/datastore_doc.go

```diff
@@ -41,7 +41,7 @@ type DataStoreKey struct {
 	FieldID          string
 }
 
-var _ Key = (*DataStoreKey)(nil)
+var _ Walkable = (*DataStoreKey)(nil)
```

Comment:
question: What inspired the name change, and the context for the name "walkable"? I am guessing it's the key that can be accessed.

EDIT: Was answered by documentation on `Walkable` ignore this question.